### PR TITLE
Added instructions for Jeager

### PR DESCRIPTION
**Pull request by MA-TATAS**
I have added instructions for Jaeger using notes that i kept while creating a jacket. I plan to add more details including more drawings sometime when i have time. I will create a different pull request for that since i cannot predict when that will be.

### CODE REVIEW

**Comment by BenJamesBen** [comment]
I created a deployment so people can view the changed web page:
https://freesewing-n8ns2fn0z-benjamesben.vercel.app/docs/patterns/jaeger/instructions

**Review by joostdecock — APPROVED** [review]

**Comment by joostdecock** [comment]
@all-contributors please add @MA-TATAS for docs

**Comment by joostdecock** [comment]
@MA-TATAS Thank you for your contribution 🙏

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 1]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+
+#### Add interfacing or canvas to the outside collar stand and under collar
+Either use a fusable interfacing, or, if you want to be hardcore use a horsehair canvas. 
+Stich it to the outside collor stand using the same technique as when adding canvas to the lapel.
+![Add canvas to the collor](collarWithCanvas.svg)
+
+#### Sew the outside collarstand to the under collar, and add to jacket
+Make sure the outside collarstand and under collar are alligend and sew together
+Sew the outside collarstand+undercollar to the neckline of the jacket
+Give small cuts in the fabric around the neckline to give it more ease
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:263]:
```suggestion
Clip the seam allowance around the neckline to give it more ease. Be careful not to clip the stitching.
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 2]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+Cut the seam allowance in the corners diagonally for a better fit
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:264]:
```suggestion
Trim the corners to reduce bulk and make it easier for turning,
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 3]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+
+![make small cuts in SA around collar](collarCuts.svg)
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:266]:
```suggestion
![clip the SA around collar](collarCuts.svg)
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 4]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+
+
+### Sleeves
+Sew the uppersleeve to the undersleeve untill the split
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:270]:
```suggestion
With *good sides together* sew the topSleeve to the underSleeve untill the split.
```

Is split the best term here? Would vent or placket be better?

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 5]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+
+#### Create the split at the eng of the sleeve
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:272]:
```suggestion
#### Create the split at the end of the sleeve
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 6]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+
+Sew the other side of the sleeves
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:274]:
```suggestion
With *good sides together* sew the other side of the sleeves
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 7]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+Turn the sleeves with the good side out
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:275]:
```suggestion
Turn the sleeves good sides out
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 8]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+Add the sleeves to the jacket
+
+
+### Inside pocket
+<note>
+The inside pocket is a double welt pocket. Which is the same as the front pockets but without a cover.
+Please see images at the front pockets for details
+</note>
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:283]:
```suggestion
### Inside pocket

<Note>
The inside pocket is a double welt pocket. Which is the same as the front pockets but without a cover.
Please see images at the front pockets for details
</Note>
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 9]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+
+Locate the position of the pocket on the left inner lining with basting
+Add some fusible interfacting around the area
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:286]:
```suggestion
Add some fusible interfacing around the area
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 10]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+Draw the pocket on the interfacing.
+Add the  welts to the upper and lower side of the pocket
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:288]:
```suggestion
Add the welts to the upper and lower side of the pocket
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 11]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+Sew the welts on the line on both sides of the pocket.
+Make use the welts end exactly on top of eachother.
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:290]:
```suggestion
Make sure the welts end exactly on top of eachother.
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 12]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+Cut the pocket entrance to about 1 or 2 cm from the end.
+Cut the last part diagonally so you get a fish bone fcut
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:292]:
```suggestion
Cut the last part diagonally so you get a fish bone cut
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 13]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+Turn the welts inside out, baste and press
+Turn the triangles inside out, baste and press
+Sew around the pocket to put everything in place
+Take the pocketbag, sew it to the lower welt and the other part to the upper welt (See: chest pocket)
+
+### Lining
+Sew the lining back seam untill the split
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:299]:
```suggestion
Sew the lining back seam until the vent
```

This one I believe is definitely vent

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 14]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+Sew the figure seams in the front parts
+Add the whole lining together (front, side, back)
+Add the from lining to the front facing. LEave about 5cm open from the bottom
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:302]:
```suggestion
Add the from lining to the front facing. Leave about 5cm open from the bottom
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 15]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+Close the shoulders
+Add the inside collar stand to the uppercollar and add to the lining jacket
+Make small cuts in the neckline and the corners
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:305]:
```suggestion
Clip the neckline and the corners
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 16]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+Add upper to lower sleeves
+Add sleeves to lining jacket
+
+### Close it up
+But the jacket + lining jackets on top of each other with the good sides.
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:310]:
```suggestion
Put the jacket + lining jackets on top of each other with the *good sides together*.
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 17]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+Sew around the collar and continue all the way to the front facing
+Turn the jacket right side out
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:312]:
```suggestion
Turn the jacket *good sides* out
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 18]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+Fold the sleeves in place (make sure they are not turned!!)
+Add the sleeves lining to the sleeves split
+Make sure the collars are straight on top of each other (they allign) and add the seam allowences to each other by hand.
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:315]:
```suggestion
Make sure the collars are straight on top of each other (they should align) and add the seam allowances to each other by hand.
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 19]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+Add in a few places front SA to lining SA by hand from the inside of the jacket
+Create the backsplit and manually add the lining to the main fabric
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:317]:
```suggestion
Create the back vent and manually add the lining to the main fabric
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 20]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+Add buttonholes and buttons
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:318]:
```suggestion
Sew buttons and buttonholes
```

File: markdown/org/docs/patterns/jaeger/instructions/en.md [hunk 21]
```diff
@@ -239,3 +239,84 @@ what you need.
...
+
+
+
+
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/patterns/jaeger/instructions/en.md:322]:
```suggestion
```

**Review by bobgeorgethe3rd — CHANGES_REQUESTED** [review]
This a first run through. It is mostly to fix typos. It will need a more extensive review for wording etc.

**Pull request merged**